pyfr/backends/openmp/kernels/base.mako

```diff
@@ -5,16 +5,13 @@
 #include <stdlib.h>
 #include <tgmath.h>
 
-#define PYFR_ALIGN_BYTES ${alignb}
 #define SOA_SZ ${soasz}
+#define SZ ${csubsz}
```

Comment:
This should be renamed to something more appropriate.

Reply:
Any suggestions?

Reply:
BLK_SZ?

Reply:
Better.